docker/rsem/Dockerfile

```diff
@@ -1,11 +1,16 @@
 FROM centos:centos7
-MAINTAINER Jishu Xu <jishuxu@broadinstitute.org>
-LABEL Description="This image is used for running RSEM RNA-Seq analysis tools "
+
+LABEL maintainer="Jishu Xu <jishuxu@broadinstitute.org>" \
+  software="RSEM" \
+  version="v1.3.0" \
+  Description="This image is used for running RSEM RNA-Seq analysis tools " \
+  website="https://github.com/deweylab/RSEM"
+
 WORKDIR /usr/local/
 RUN yum groupinstall -y 'Development Tools' 
 RUN yum install -y perl-Env 
 RUN yum install -y zlib-devel
-RUN git clone https://github.com/deweylab/RSEM.git
+RUN git clone --depth 1 --branch v1.3.0 https://github.com/deweylab/RSEM.git
```

Comment:
This syntax is a bit confusing -- I thought at first glance that you were checking out a branch called v1.3.0, but I see that no such branch exists and --branch can take a tag name instead, which is what is being used in this case.

In addition to being a bit confusing, there is a slight danger that someone could create a branch in this repo called v1.3.0 and then this changes what we're pulling in here.

I prefer this syntax:
```
git clone https://github.com/deweylab/RSEM.git
git checkout tags/v1.3.0
```

I realize that this checks out more stuff, but I'm sure it's a small amount of data compared to the overall image size.

Reply:
Thanks for this comment, which is just I want regarding this change! I thought about the trade-offs between `git clone --depth 1 --branch v1.3.0 https://github.com/deweylab/RSEM.git
` and `git clone https://github.com/deweylab/RSEM.git
git checkout tags/v1.3.0` I chose the former one just because it checks out less stuff. But yes it may be danger in the future and at the same time, it is relatively small compared to the overall image size. I will change it.